src/server/game/Entities/Unit/UnitDefines.h

```diff
@@ -152,7 +152,7 @@ enum UnitFlags : uint32
     UNIT_FLAG_UNK_28                = 0x10000000,
     UNIT_FLAG_UNK_29                = 0x20000000,           // used in Feing Death spell
     UNIT_FLAG_SHEATHE               = 0x40000000,
-    UNIT_FLAG_UNK_31                = 0x80000000,
+    UNIT_FLAG_HAS_IMMUNITY_AURA     = 0x80000000,
```

Comment:
Not researched this on later versions but it might be used for more things than just auras, better to simply call it "IMMUNE"